tasks/orchestrator.yml

```diff
@@ -16,3 +16,24 @@
   register: yaml_client_config_secret
   tags:
   - oc
+
+# Setup imagestream
+- name: copy imagestream
+  template:
+    src: osbs-buildroot-imagestream.yml.j2
+    dest: "{{ osbs_openshift_home }}/{{ inventory_hostname }}-{{ osbs_namespace }}-osbs-buildroot-imagestream.yml"
+  environment: "{{ osbs_environment }}"
+  register: yaml_imagestream
+  tags:
```

Comment:
Do we only need to do these steps when osbs_buildroot_repository is set?

Reply:
ahh right, now that it is variable it makes sense